kano_world/session.py

```diff
@@ -526,8 +543,6 @@ def _process_notification(self, entry):
             '/images/notification/280x170/share-pong.png'
         PONG_SHARE_IMG = media_dir + \
             '/images/notification/280x170/share-minecraft.png'
-        SP_IMG = media_dir + \
```

Comment:
Good point. It was like that before. Since you have pointed it out, I will change it :-)